proto/poktroll/migration/event.proto

```diff
@@ -29,12 +29,33 @@ message EventMorseAccountClaimed {
   // The height (on Shannon) at which the claim was executed (i.e. claimed).
   int64 claimed_at_height = 1 [(gogoproto.jsontag) = "claimed_at_height"];
 
-  // The balance which was claimed.
+  // The total balance which was claimed.
   cosmos.base.v1beta1.Coin claimed_balance = 2 [(gogoproto.jsontag) = "claimed_balance", (gogoproto.nullable) = false];
 
   // The bech32-encoded address of the Shannon account to which the claimed balance will be minted.
   string shannon_dest_address = 3 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.jsontag) = "shannon_dest_address"];
 
   // The hex-encoded address of the Morse account whose balance will be claimed.
   string morse_src_address = 4 [(gogoproto.jsontag) = "morse_src_address"];
-}
\ No newline at end of file
+}
+
+// EventMorseApplicationClaimed is emitted when a MorseAccount is claimed on-chain as a staked application.
+message EventMorseApplicationClaimed {
```

Comment:
I think we should include the staked application in this event; this would be consistent with normal application staking behavior.